pages/instructorIssues/instructorIssues.ejs

```diff
@@ -170,9 +170,15 @@
               <% } else { %>
               <i class="fa fa-check-circle text-success issue-status-icon"></i>
               <% } %>
+              <% if (row.instance_question_id) { %>
+              <a class="d-block" href="<%= plainUrlPrefix %>/course_instance/<%= row.course_instance_id %>/instance_question/<%= row.instance_question_id %>/?variant_id=<%= row.variant_id %>">
```

Comment:
https://github.com/PrairieLearn/PrairieLearn/issues/3549#issuecomment-750345502

> I'm sorry to only now comment on this after you already implemented the other option in #3568, but I actually think it would be conceptually cleaner to fix the issue with the idea above. **That is, link to the student view of the question**. This way we can keep "instructor view of question" and "student view of question" completely separate. I worry that it's confusing to show the instructor view of a question but with a link to a student's assessment instance. I also think it's conceptually cleaner to link from the issue to the page that the student was on when they filed the issue.

Reply:
Emphasis added.

Reply:
And yes, it supports variant_id (this is linked to from the assessment log, when selecting a specific instance (e.g., S-A1.5#1).

Reply:
In any case, switched back to the instructor view, so the link has a consistent destination. The student view is now added to the dropdown on the side.